src/components/Header/Pages.js

```diff
@@ -1,6 +1,6 @@
 import strings from 'lang';
```

Comment:
No, there is Navbar component which is using this. I left the name as `Pages` because it is nested under `Header` and thus should be okay.